spring-cloud-stream/src/main/java/org/springframework/cloud/stream/annotation/EnableModule.java

```diff
@@ -47,7 +46,7 @@
 @Configuration
 @MessageEndpoint
 @Import({RedisServiceConfiguration.class, RabbitServiceConfiguration.class,
-	ChannelBindingAdapterConfiguration.class, CodecConfiguration.class, ChannelBindingAdapterRunner.class,
```

Comment:
Maybe auto configuration via {{spring.factories}} ? I'm a little wary about that one because of what it does when multiple binder implementations are on the CP, but it's no different than this, with the added advantage of not decluttering {{@EnableModule}}.